folium/features.py

```diff
@@ -901,6 +937,14 @@ def __init__(self, icon_image, icon_size=None, icon_anchor=None,
         self.shadow_size = shadow_size
         self.shadow_anchor = shadow_anchor
         self.popup_anchor = popup_anchor
+        self.options = {
```

Comment:
Same as for divIcon. Also, `LatLngPopup` currently doesn't accept options.

Reply:
Roger.